### PR TITLE
Alcail/call composite min max

### PR DESCRIPTION
**Pull request by alcail**
# What
added min width/height to Calling composite(global and by page)
* Mobile min is 312px x 336px (19.5rem x 21rem), knowing iPhone SE is 320px x 568px
image

* Desktop min is 480px x 352 px (30rem x 22rem)
image

# Why
no min were defined for our Calling composite
task https://skype.visualstudio.com/SPOOL/_workitems/edit/2630302

# How Tested
ran Call sample and played with min height/width on desktop/mobile configuration

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by JamesBurnside — APPROVED** [review]

**Comment by anjulgarg** [comment]
@alcail 
packages/react-composites/tests/browser/meeting/MeetingComposite.test.ts-snapshots/meeting-gallery-screen-with-people-pane-Mobile-Android-Portrait-linux.png

doesn't look right

**Review by JamesBurnside — CHANGES_REQUESTED** [review]
Resetting vote while snapshots are looked at

**Comment by alcail** [comment]
> Resetting vote while snapshots are looked at (e.g. call config screen title no longer centered: image )

@JamesBurnside , yep, saw that and reverted th Stack to Text change and will revisit that later as it's not part of this PR ;)

**Comment by JamesBurnside** [comment]
Any idea why this spacing changed? /is it intended?
image

**Comment by alcail** [comment]
> Any idea why this spacing changed? /is it intended? image

to avoid this issue of parent not being high enough
image

**Comment by JamesBurnside** [comment]
> > Any idea why this spacing changed? /is it intended?
> 
> to avoid this issue of parent not being high enough
> 
> I went from 3.875rem to 4rem 

Ideally we don't set any height and stacks should correctly size everything, did some digging and its because we use dockedBottom as our ControlBar layout which has position:absolute and thus the stack considers the space it should take up to be 0 so doesn't do the automatic sizing. I'll do a subsequent PR after this one that fixes that issue as the changes are fairly significant.

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-composites/src/composites/CallComposite/CallComposite.tsx [hunk 1]
```diff
@@ -146,12 +147,14 @@ export const CallComposite = (props: CallCompositeProps): JSX.Element => {
   return (
     <BaseComposite {...props}>
       <CallAdapterProvider adapter={adapter}>
-        <MainScreen
-          callInvitationUrl={callInvitationUrl}
-          onFetchAvatarPersonaData={onFetchAvatarPersonaData}
-          onFetchParticipantMenuItems={onFetchParticipantMenuItems}
-          options={options}
-        />
+        <div className={props.options?.mobileView ? mainScreenContainerStyleMobile : mainScreenContainerStyleDesktop}>
```
Comment (alkwa-msft) [inline packages/react-composites/src/composites/CallComposite/CallComposite.tsx:150]: minor nit. can we determine the value of classname into a variable and just use the variable in this prop when we are returning our JSX.element?

File: packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx [hunk 2]
```diff
@@ -28,14 +29,21 @@ export interface CallArrangementProps {
...
 /**
  * @private
  */
 export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
   return (
-    <Stack horizontalAlign="center" verticalAlign="center" styles={containerStyles} grow data-ui-id={props.dataUiId}>
+    <Stack
+      horizontalAlign="center"
+      verticalAlign="center"
+      className={props.mobileView ? containerStyleMobile : containerStyleDesktop}
```
Comment (alkwa-msft) [inline packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:43]: minor nit. can we determine the value of classname into a variable and just use the variable in this prop when we are returning our JSX.element?

**Review by alkwa-msft — APPROVED** [review]

**Pull request merged**